e2e/utils/waits-utils.ts

```diff
@@ -263,7 +263,7 @@ export async function waitForText(page: Page,
  */
 export async function waitWhileLoading(page: Page, timeOut: number = 90000): Promise<void> {
   const notBlankPageSelector = '[data-test-id="sign-in-container"], title:not(empty), div.spinner, svg[viewBox]';
-  const spinElementsSelector = '[style*="running spin"], .spinner:empty, [style*="running rotation"]';
+  const spinElementsSelector = '[style*="running spin"], .spinner:empty, svg[style*="running rotation"]:not([aria-hidden="true"])';
```

Comment:
exclude a recently added spinner. 
<img width="115" alt="Screen Shot 2020-10-27 at 10 48 53 AM" src="https://user-images.githubusercontent.com/35533885/97318280-1427e880-1842-11eb-8f44-970286ea61b6.png">


Test failed in https://app.circleci.com/pipelines/github/all-of-us/workbench/5308/workflows/0b6cbff1-128e-49da-a3d4-d53c4786f7f1/jobs/106906/tests

stacktrace:
```
FAIL  tests/cohorts/cohorts-create.spec.ts (152.837 s)
  User can create new Cohorts
    ✕ Add Cohort of Physical Measurements BMI (107149 ms)
    ○ skipped Add Cohort of EKG condition with modifiers

  ● User can create new Cohorts › Add Cohort of Physical Measurements BMI

    TimeoutError: waiting for function failed: timeout 90000ms exceeded

      273 | 
      274 |   // Wait for spinners stop and gone.
    > 275 |   await page.waitForFunction((css) => {
          |              ^
      276 |     const elements = document.querySelectorAll(css);
      277 |     return elements && elements.length === 0;
      278 |   }, {polling: 'mutation', timeout: timeOut}, spinElementsSelector);

      at new WaitTask (node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:394:34)
      at DOMWorld.waitForFunction (node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:316:16)
      at Frame.waitForFunction (node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:869:32)
      at Page.waitForFunction (node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:1221:33)
      at Object.waitWhileLoading (utils/waits-utils.ts:275:14)
```